### PR TITLE
Rename ubuntu-touch-sounds

### PR DESCRIPTION
**Pull request by tari01**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -33,7 +33,7 @@ Build-Depends: cmake,
                gvfs-daemons <!nocheck>,
                systemd [linux-any],
 # for phone alarm/calendar notification sound tests:
-               ubuntu-touch-sounds | hello,
+               lomiri-touch-sounds | hello,
```
Comment (sunweaver) [inline debian/control:36]: Maybe lomiri-sounds? Let's coordinate this first...

**Comment by tari01** [comment]
Related/affected:

https://gitlab.com/ubports/core/ubuntu-touch-sounds/-/merge_requests/5
https://gitlab.com/ubports/core/lomiri-system-settings/-/merge_requests/315

**Review by sunweaver — APPROVED** [review]
LGTM, waiting for today's devsync to coordinate the new package name.

**Comment by peat-psuwit** [comment]
So the default sounds doesn't become compile-time options?

**Comment by sunweaver** [comment]
> So the default sounds doesn't become compile-time options?

Do we need that? Maybe a good idea? Still, what do we want to use as the very default?

File: debian/control (hunk 1 above)
Comment (mariogrip) [reply debian/control:36]: Lomiri-sound sounds better imo

**Comment by UniversalSuperBox** [comment]
Counter-proposal: I'll make a new PR series to rename ubuntu-touch-sounds to lomiri-sounds, move all of the sounds to `/usr/share/sounds/lomiri/`, and create two new packages:

* `lomiri-sounds-ubuntu-compat`, which creates `/usr/share/sounds/ubuntu/ -> /usr/share/sounds/lomiri/` and `/usr/share/sounds/ubports/ -> /usr/share/sounds/lomiri/` links.
* `ubuntu-touch-sounds`, a transition package which depends upon `lomiri-sounds` and `lomiri-sounds-ubuntu-compat`

**Comment by peat-psuwit** [comment]
> Do we need that?

I mean, this kind of things are distribution's identity, and therefore should be configurable. Also, for a similar reason I feel like renaming ubuntu-touch-sound isn't really appropriate. It's Ubuntu Touch's identity, and should stay with Ubuntu Touch (and the trademark agreement might be involved, although don't ask me on that).

> Still, what do we want to use as the very default?

I would say, something from `/usr/share/sounds/{freedesktop,sound-icons}`, which feels distro-agnostic enough. IMO `freedesktop/stereo/alarm-clock-elapsed.oga` might work for alarm, and `sound-icons/xylofon.wav` for calander. Or, if one prefers single package, `sound-icons/cembalo-6.wav` probably works for alarm.

**Comment by UniversalSuperBox** [comment]
Hey, that's probably simpler. The freedesktop sounds are available on [Debian](https://packages.debian.org/stretch/sound/sound-theme-freedesktop), [Ubuntu](https://packages.ubuntu.com/impish/sound-theme-freedesktop), and [Arch](https://archlinux.org/packages/extra/any/sound-theme-freedesktop/) at least. 

Looks like the only required change is https://github.com/AyatanaIndicators/ayatana-indicator-datetime/blob/b3e4cbdde1444b134d8b2729f3abcee405ca1241/CMakeLists.txt#L46-L47?

**Comment by tari01** [comment]
Alright, I'll also add my voice to this one. We should keep in mind that Lomiri is the only client that will use sounds here. The freedesktop sounds are not needed.

**Comment by peat-psuwit** [comment]
> We should keep in mind that Lomiri is the only client that will use sounds here.

Think when Lomiri goes into Debian. IMO it's not appropriate to package `ubuntu-touch-sound` into Debian, and renaming it to `lomiri-sound` doesn't make it more appropriate. What do you think?

**Comment by tari01** [comment]
> > We should keep in mind that Lomiri is the only client that will use sounds here.
> 
> Think when Lomiri goes into Debian. IMO it's not appropriate to package `ubuntu-touch-sound` into Debian, and renaming it to `lomiri-sound` doesn't make it more appropriate. What do you think?

Well, my reason dictates to simply rename it. On the other hand, I know absolutely nothing about customs and ethics in the FOSS world, so I'll leave the final decision to the those who commented above.

**Comment by sunweaver** [comment]
@UniversalSuperBox @peat-psuwit Question is, if Lomiri in Debian (and elsewhere) shall use the same sounds as it does on Ubuntu Touch. If so, the ubuntu sound theme needs to be forked and used as lomiri-sounds. If not, then all of Lomiri should (IMHO) default to freedesktop.org sounds. Don't you think?

**Comment by sunweaver** [comment]
> @UniversalSuperBox @peat-psuwit Question is, if Lomiri in Debian (and elsewhere) shall use the same sounds as it does on Ubuntu Touch. If so, the ubuntu sound theme needs to be forked and used as lomiri-sounds. If not, then all of Lomiri should (IMHO) default to freedesktop.org sounds. Don't you think?

On Ubuntu Touch, Lomiri then can be tweaked to use Ubuntu Touch sounds, but that feels like quite a hassle.

**Comment by gber** [comment]
Just ran into this with the telephony-service binary package which also depends on this.

**Comment by peat-psuwit** [comment]
> Question is, if Lomiri in Debian (and elsewhere) shall use the same sounds as it does on Ubuntu Touch.

I would say, no. But @UniversalSuperBox should know better about this stuff.

**Comment by UniversalSuperBox** [comment]
I'm working on the transitional package, and a correct rename of lomiri-sounds, now.

**Comment by UniversalSuperBox** [comment]
https://gitlab.com/ubports/core/ubuntu-touch-sounds/-/merge_requests/7 relevant to this PR
https://gitlab.com/ubports/core/ubuntu-touch-sounds-compat/-/merge_requests/1 relevant to being able to merge that MR

File: debian/control (hunk 1 above)
Comment (sunweaver) [reply debian/control:36]: @tari01: I will change this to lomiri-sounds and then push and merge.

**Review by sunweaver — APPROVED** [review]

**Pull request merged**